platform/source/mbed_os_timer.cpp

```diff
@@ -16,9 +16,9 @@
  */
 
 #include "platform/mbed_power_mgmt.h"
-#include "platform/source/mbed_os_timer.h"
 #include "platform/CriticalSectionLock.h"
-#include "platform/source/SysTimer.h"
+#include "source/SysTimer.h"
+#include "source/mbed_os_timer.h"
 #include "us_ticker_api.h"
 #include "lp_ticker_api.h"
 #include "mbed_critical.h"
```

Comment:
How do you feel about the following in this same commit:
```suggestion
```

Reply:
I rather do this separately, to get this in as it is.